src/pages/register/index.tsx

```diff
@@ -109,7 +125,16 @@ export const Register = () => {
             Remember me
           </label>
         </div>
-        <Button size="fl">Next</Button>
+        <Button
+          disabled={isSignUpLoading}
+          size="fl"
+          onClick={() => handleRegisterFunc()}
+        >
+          Create Account
+        </Button>
+        <div className="w-full">
+          {error ? <p className="mt-2.5 text-red">Something went wrong</p> : null}
```

Comment:
We need to show error from BE if it's there

Reply:
Done